Makefile

```
@@ -48,6 +48,7 @@ MLRUN_RAISE_ON_ERROR ?= true
 MLRUN_SKIP_CLONE ?= false
 MLRUN_RELEASE_NOTES_OUTPUT_FILE ?=
 MLRUN_SYSTEM_TESTS_CLEAN_RESOURCES ?= true
+MLRUN_SYSTEM_TEST_MARKERS ?=
```

```suggestion
MLRUN_SYSTEM_TEST_MARKERS ?= "not enterprise"
```
and then you don't need the if in
```
-m $(if $(MLRUN_SYSTEM_TEST_MARKERS),"$(MLRUN_SYSTEM_TEST_MARKERS)","not enterprise") \
```
it can be
```
-m $(MLRUN_SYSTEM_TEST_MARKERS) \
```
although it's not exactly the same because then users can set MLRUN_SYSTEM_TEST_MARKERS="" if we don't want to allow it

---

well, I agree we need to ensure that some of the markers are not given to the developers and are run regardless. Id take it on a followup pr
regarding the variable, I thought it could be served for the enterprise system test afterwards, and thus I did not set a global default but rather a test-specific one